### PR TITLE
Hotfix: Old code present!

### PR DESCRIPTION
**Pull request by AndreasWebdev**
Please consider a hotfix @torch2424 as I noticed that the changeConfig function still had a like of old code breaking the function/changeconfig API call

### CODE REVIEW

File: src/api/config.js [hunk 1]
```diff
@@ -3,20 +3,20 @@ const upath = require('upath');
 const fs = require('fs-extra');
 const editJsonFile = require('edit-json-file');
 
-const getFullConfig = async (path) => {
+const getFullConfig = async path => {
```
Comment (AndreasWebdev) [inline src/api/config.js]: I am super confused why github is removing these.. it appearently is valid js though

Comment (torch2424) [reply src/api/config.js]: Oh its from the linter, I have github hooks set up so the linter runs on commit 👍

**Comment by AndreasWebdev** [comment]
The config.js overrides shouldn't factor in here, as we are only editing the config.json variables. It just had a line that referenced the old method of getting a config from a key that appearently didn't get deleted. It's ready to deploy!

**Comment by torch2424** [comment]
Awesome thanks! 👍 I'll deploy this once I get home, since I'm not signed into my personal npm account at work 😢

**Comment by torch2424** [comment]
Also, you got a twitter or discord? That way we can chat on a medium other than github issues and PRs? Haha! Would be good for planning and things like this 👍

**Comment by AndreasWebdev** [comment]
I actually tried inviting you to gitter.im for a repo-chat, however I think I would be more active on either Twitter (@iKlikla) or Discord (iKlikla#4975)

**Comment by torch2424** [comment]
Oh haha! I don't use gitter, But I totally can! I'll reach out over discord :)

**Pull request merged**